docs/src/css/components/_combobox.scss

```
@@ -1,58 +1,25 @@
 ix-combobox {
   position: relative;
-  display: inline-block;   
-  width: 260px;           
-}
-
 
+  display: inline-flex;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)